docs/ovms_cloud_native_quickstart.md

```diff
@@ -0,0 +1,298 @@
+```
+
+- 2. Verify the OpenVINO Model Server from the client test pod
+
+```bash
+$ kubectl exec -it $(kubectl get pod -o jsonpath="{.items[0].metadata.name}" -l app=client-test) -- bash
+$ curl http://model-server-sample-ovms:8000/v1/config
+{
+  "faster_rcnn": {
+    "model_version_status": [
+      {
+        "version": "1",
+        "state": "AVAILABLE",
+        "status": {
+          "error_code": "OK",
+          "error_message": "OK"
+        }
+      }
+    ]
+  }
+}
+```
+
+Above output indicate that the model server is working well.
+
+- 3. Running inference inside the client test pod 
+
+Prepare the Example Client Components in local file system:
+
+```bash
+$ mkdir -p ~/ecc && cd ~/ecc
+$ wget https://raw.githubusercontent.com/openvinotoolkit/model_server/main/demos/object_detection/python/object_detection.py
+$ wget https://raw.githubusercontent.com/openvinotoolkit/model_server/main/demos/object_detection/python/requirements.txt
+$ wget https://raw.githubusercontent.com/openvinotoolkit/open_model_zoo/master/data/dataset_classes/coco_91cl.txt
+$ wget https://storage.openvinotoolkit.org/repositories/openvino_notebooks/data/data/image/coco_bike.jpg
+```
+
+Copy the Example Client Components into client-test pod:
+
+```bash
+$ kubectl cp -n default . $(kubectl get pod -o jsonpath="{.items[0].metadata.name}" -l app=client-test):/root
+$ kubectl exec -it $(kubectl get pod -o jsonpath="{.items[0].metadata.name}" -l app=client-test) -- ls -ls /root
+  4 -rw-r--r-- 1 1000 1000    702 Apr 10 09:03 coco_91cl.txt
+184 -rw-r--r-- 1 1000 1000 186584 Apr 10 09:03 coco_bike.jpg
+  8 -rw-r--r-- 1 1000 1000   6182 Apr 10 09:03 object_detection.py
+268 -rw-r--r-- 1 root root 271313 Apr 11 07:10 output.jpg
+  4 -rw-r--r-- 1 1000 1000     44 Apr 10 09:03 requirements.txt
+```
+
+Go to the folder with the Example Client Components and install dependencies:
+
+```bash
+$ kubectl exec -it $(kubectl get pod -o jsonpath="{.items[0].metadata.name}" -l app=client-test) -- bash
+$ cd /root
+$ pip install --upgrade pip
+$ pip install -r requirements.txt
+
+$ python3 object_detection.py --image coco_bike.jpg --output output.jpg --service_url model-server-sample-ovms:9000
+$ ls -ls
+total 464
+  4 -rw-r--r-- 1 1000 1000    702 Apr 10 09:03 coco_91cl.txt
+184 -rw-r--r-- 1 1000 1000 186584 Apr 10 09:03 coco_bike.jpg
+  8 -rw-r--r-- 1 1000 1000   6182 Apr 10 09:03 object_detection.py
+264 -rw-r--r-- 1 root root 270070 Apr 11 10:08 output.jpg
+  4 -rw-r--r-- 1 1000 1000     44 Apr 10 09:03 requirements.txt
+```
+
+> **Note**: - Please make sure the `HTTP_PROXY` and `HTTPS_PROXY` are set correctly to install the dependencies libs in `requirements.txt`.
+
+- 4. Review the Results
+
+Showing as previous instruction, User can find output file named `output.jpg` containing inference results. In this case, it should be a modified input image with bounding boxes indicating detected objects and their labels.
+
+![Inference results](cloud_native_quickstart_result.jpeg)
```

Comment:
Image is missing in this directory

Reply:
I have uploaded the image, done!  Thanks!